packages/selinux-policy/rules.cil

```diff
@@ -155,9 +155,9 @@
 ; files and directories on /local.
 (allow control_s state_t (files (mutate mount)))
 
-; Untrusted subjects cannot modify "state" or "secret" files.
-(neverallow untrusted_s state_t (files (mutate mount)))
-(neverallow untrusted_s secret_t (files (mutate mount)))
+; Unprivileged subjects cannot modify "state" or "secret" files.
+(neverallow unprivileged_s state_t (files (mutate mount)))
+(neverallow unprivileged_s secret_t (files (mutate mount)))
```

Comment:
True of untrusted as well, right?  It might help to clarify if we add neverallow for untrusted_s where we have it for unprivileged_s, except in cases where they intentionally differ...

Reply:
`control_t` is part of `untrusted_s` , but needs to be able to modify these files so that non-superpowered host containers can write to their own rootfs.